x-pack/plugins/ingest_manager/common/constants/agent_policy.ts

```diff
@@ -3,15 +3,22 @@
  * or more contributor license agreements. Licensed under the Elastic License;
  * you may not use this file except in compliance with the Elastic License.
  */
-import { AgentPolicyStatus, DefaultPackages } from '../types';
-
+import { AgentPolicy, DefaultPackages } from '../types';
 export const AGENT_POLICY_SAVED_OBJECT_TYPE = 'ingest-agent-policies';
 
-export const DEFAULT_AGENT_POLICY = {
+export const agentPolicyStatuses = {
+  Active: 'active',
```

Comment:
nitpick and personal preference but I prefer having constant uppercase like `AGENT_POLICY_STATUSES.ACTIVE`